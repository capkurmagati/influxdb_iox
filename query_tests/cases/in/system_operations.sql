-- test of system.operations
-- IOX_SETUP: TwoMeasurementsManyFieldsLifecycle

SELECT
  id,
  status, CAST(cpu_time_used AS BIGINT) > 0 as took_cpu_time,
  CAST(wall_time_used AS BIGINT) > 0 as took_wall_time,
  partition_key,
  chunk_id,
  description
FROM
  system.operations;
