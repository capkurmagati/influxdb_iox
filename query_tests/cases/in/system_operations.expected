-- Test Setup: TwoMeasurementsManyFieldsLifecycle
-- SQL: SELECT id, status, CAST(cpu_time_used AS BIGINT) > 0 as took_cpu_time, CAST(wall_time_used AS BIGINT) > 0 as took_wall_time, partition_key, chunk_id, description FROM system.operations;
+----+----------+---------------+----------------+---------------+----------+---------------------------------+
| id | status   | took_cpu_time | took_wall_time | partition_key | chunk_id | description                     |
+----+----------+---------------+----------------+---------------+----------+---------------------------------+
| 0  | Complete | true          | true           | 1970-01-01T00 | 0        | Loading chunk to ReadBuffer     |
| 1  | Complete | true          | true           | 1970-01-01T00 | 0        | Writing chunk to Object Storage |
+----+----------+---------------+----------------+---------------+----------+---------------------------------+
